src/backend/dx11/src/device.rs

```diff
@@ -1614,7 +1644,7 @@ impl hal::Device<Backend> for Device {
                 StorageTexelBuffer | StorageBuffer |
                 InputAttachment | StorageBufferDynamic |
                 StorageImage => {
-                    binding.handle_offset = num_s + num_t + num_u + u;
+                    binding.handle_offset = num_s + num_t + num_c + u;
```

Comment:
generally, a small fix like this would go as a separate commit, to have this one strictly as a feature commit
I don't mind mixing it at this point, when things are still in flux